screen-wake-lock/wakelock-released.https.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<link rel="help" href="https://w3c.github.io/screen-wake-lock/#dom-wakelocksentinel-released">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+promise_test(async t => {
+  await test_driver.set_permission({name: 'screen-wake-lock'}, 'granted', false);
+
+  const lock = await navigator.wakeLock.request("screen");
+  assert_false(lock.released, "lock.release must be false on creation");
+
+  return new Promise(resolve => {
+    lock.onrelease = resolve;
+    lock.release();
+    assert_false(lock.released, "WakeLockSentinel.release does not become true immediately")
+  }).then(event => {
+    assert_true(lock.released, "lock.release must be true after calling release()");
+  });
```

Comment:
```suggestion
  const p = new EventWatcher(t, screenWakeLock, "release").wait_for("release");
  lock.release();
  assert_false(lock.released, "WakeLockSentinel.release does not become true immediately")
  await p;
  assert_true(lock.released, "lock.release must be true after calling release()");
```

Reply:
Thanks, I've switched this test to using `EventWatcher`.